libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1097,6 +1099,29 @@ class MapboxNavigation @VisibleForTesting internal constructor(
         )
     }
 
+    /**
+     * start
+     */
+    fun startCopilot() {
```

Comment:
```suggestion
    fun startCopilot() = apply {
```

IMO it is nice to be able to chain functions